scripts/entity/components/movement_components/player_movement_component.gd

```diff
@@ -4,58 +4,96 @@ class_name PlayerMovementComponent extends BaseMovementComponent
 ## The max horizontal speed the entity can move.
 @export var MAX_SPEED: int = 200
 ## The jump velocity of the entity.
-@export var JUMP_VELOCITY: int = -300
+@export var JUMP_VELOCITY: int = -350
```

Comment:
This should just be a positive number, and made negative in the math it's used in

Reply:
Used _ready to invert